capture/scopes/chipwhisperer/husky.py

```diff
@@ -0,0 +1,106 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import time
+
+import chipwhisperer as cw
+import numpy as np
+from scopes.chipwhisperer.cw_segmented import CwSegmented
+
+SAMPLING_RATE_MAX = 200e6
+
+
+class Husky:
+    def __init__(self, scope_gain, num_cycles, num_segments, offset_cycles,
+                 pll_frequency, target_clk_mult):
+        self.scope = None
+        self.scope_gain = scope_gain
+        # In our setup, Husky operates on the PLL frequency of the target and
+        # multiplies that by an integer number to obtain the sampling rate.
+        # Note that the sampling rate must be at most 200 MHz.
+        self.clkgen_freq = pll_frequency
+        self.adc_mul = int(SAMPLING_RATE_MAX // pll_frequency)
+        self.sampling_rate = self.clkgen_freq * self.adc_mul
+
+        # The target runs on the PLL clock but uses internal clock dividers and
+        # multiplier to produce the clock of the target block.
+        self.target_freq = pll_frequency * target_clk_mult
```

Comment:
So we don't forget to checkt: In the previous form there was an issue here where the actual target freq. seemed different from what is computed here

Reply:
the root issue was that the target_clk_mult does not match what is happening on FPGA, see https://github.com/lowRISC/ot-sca/pull/203/files#r1385486125

Reply:
As discussed previously, we need to check with @vogelpi which `target_clk_mult` was used and where to find this parameter.